bookkeeper-server/src/main/java/org/apache/bookkeeper/zookeeper/ZooKeeperClient.java

```diff
@@ -229,7 +229,7 @@ public ZooKeeperClient build() throws IOException, KeeperException, InterruptedE
 
             if (null == connectRetryPolicy) {
                 connectRetryPolicy =
-                        new BoundExponentialBackoffRetryPolicy(sessionTimeoutMs, sessionTimeoutMs, Integer.MAX_VALUE);
+                        new BoundExponentialBackoffRetryPolicy(0, 0, Integer.MAX_VALUE);
```

Comment:
can we add a comment explaining this choice ?
this way in the future it will be easier to not fall into a question like "why are we using an hardcoded 0 value" ?

Reply:
I will add it.

Reply:
Done.